pkg/updatechecker/updatechecker.go

```diff
@@ -292,9 +404,10 @@ func CheckForUpdates(opts CheckForUpdatesOpts) (*UpdateCheckResponse, error) {
 	}
 
 	// this is to avoid a race condition where the UI polls the task status before it is set by the goroutine
-	status := fmt.Sprintf("%d Updates available...", ucr.AvailableUpdates)
-	if err := store.SetTaskStatus("update-download", status, "running"); err != nil {
```

Comment:
These store functions should still be used.  Maybe change key name so it doesn't collide for different apps.